libs/s25main/factories/BuildingFactory.cpp

```diff
@@ -43,8 +44,19 @@ noBuilding* BuildingFactory::CreateBuilding(GameWorldBase& gwg, const BuildingTy
         default: bld = new nobUsual(type, pt, player, nation); break;
     }
     gwg.SetNO(pt, bld);
+    GamePlayer &playerObject = gwg.GetPlayer(player);
     // Don't do this in ctor as building might not be fully initialized yet
-    gwg.GetPlayer(player).AddBuilding(bld, type);
+    playerObject.AddBuilding(bld, type);
+    const auto &description = BLD_WORK_DESC[type];
+    // We're filtering out JOB_PRIVATE because military buildings do their own
```

Comment:
Put on 1 line

Reply:
Is there documentation on the maximum line length acceptable in comments in this project?

Reply:
There is a .clang-format file and a `make clangFormat` target so don't worry about it, just don't manually break it and run the target. Length should be 120 or so.